install.sh

```diff
@@ -20,3 +20,8 @@ fi
 pip install -r requirements.txt
 
 . download-api-raml.sh
+
+cd schema/api-raml
```

Comment:
I think that here you are testing with a branch of the api-raml, hence the need for recompilation; however this will be run on `lax--*` instances where there is no npm so it would need to be removed. Maybe it can be kept as a separate script for development, `install-api-raml-branch.sh`.